ft_hangouts/ft_hangouts/UserStories/OverviewContact/OverviewContactViewController.swift

```diff
@@ -217,6 +212,12 @@ final class OverviewContactViewController: UIViewController, MFMessageComposeVie
     
     private func setUpDeleteButton() {
         mainStackView.addArrangedSubview(deleteButton)
+        NSLayoutConstraint.activate([
+//            deleteButton.topAnchor.constraint(equalTo: buttonsStackView.bottomAnchor, constant: 32),
+//            deleteButton.bottomAnchor.constraint(equalTo: backgroundView.bottomAnchor, constant: -32),
+//            
```

Comment:
commented code should not be in the diff

Reply:
Cleaned up